testsuite/elytron/src/test/resources/logging.properties

```diff
@@ -0,0 +1,31 @@
+# Additional loggers to configure (the root logger is always configured)
+loggers=org.wildfly.security,org.jboss.remoting3
+
+logger.level=INFO
+logger.handlers=FILE,CONSOLE
```

Comment:
IMO, if not both, then at least `CONSOLE`, the redirection to file is then controlled by surefire plugin.